.travis.yml

```diff
@@ -1,20 +1,30 @@
 ---
 language: csharp
 
-mono:
- - 4.6.1
- - latest # => "stable release"
- - weekly # => "latest commits"
+matrix:
```

Comment:
I don't think it's important, that's why I removed it. Would you like me to bring it back? Is it really so worthwhile? Why?